Visualization/Interpolation/.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
delete

Reply:
Rename files, this naming is hard to follow.